src/command.ts

```diff
@@ -383,7 +383,7 @@ export namespace Command {
     run(argv?: string[], config?: LoadOptions): Promise<any>
   }
 
-  export interface Loadable extends Cached {
+  export type Loadable = Cached & {
```

Comment:
I'd like a jsdoc explanation of what this type is meant to be used for, since lots of function params refer to it.